ipyleaflet/leaflet.py

```diff
@@ -814,7 +822,15 @@ class Heatmap(RasterLayer):
     max = Float(1.0).tag(sync=True, o=True)
     radius = Float(25.0).tag(sync=True, o=True)
     blur = Float(15.0).tag(sync=True, o=True)
-    gradient = Dict({0.4: 'blue', 0.6: 'cyan', 0.7: 'lime', 0.8: 'yellow', 1.0: 'red'}).tag(sync=True, o=True)
+    colors = ['blue', 'cyan', 'lime', 'yellow', 'red']
+    values = [0.4, 0.6, 0.7, 0.8, 1.0]
```

Comment:
You might want to use traits for these, otherwise they are not configurable.

Reply:
Logics of the attributes has been changed a little bit in [93fa3c4](https://github.com/jupyter-widgets/ipyleaflet/pull/1036/commits/93fa3c40e40500cb386fa9fd645d01f2415bd2b5), but the choice of the values and colors are configurable, as shown in the the example notebook Heatmap_with_colormap. The user can select the values and the colors.